addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -519,6 +519,11 @@
 		{
 			"prefab"		"159"
 		}
+		"425"	//The Family Business
+		{
+			"desp"			"The Family Business: {positive}speed boost on hit, {negative}50% slower reload speed"
```

Comment:
`The` in weapon name can drop out like every other weapons, also `Speed boost` with S caps lock